source/macos/AdaptiveCards/AdaptiveCards/Views/ACRView.swift

```diff
@@ -31,6 +32,12 @@ class ACRView: ACRColumnView {
         super.init(coder: coder)
     }
     
+    func addShowCard(_ cardView: ACRView, parentView: ACRView) {
+        cardView.parent = parentView
```

Comment:
```suggestion
    func addShowCard(_ cardView: ACRView) {
        cardView.parent = self
```
I think the parentParam is not required. It's just gonna be `self` always.

Reply:
Right, makes sense!